photon-ml/src/integTest/scala/com/linkedin/photon/ml/util/PhotonLoggerIntegTest.scala

```diff
@@ -62,14 +62,14 @@ class PhotonLoggerIntegTest extends SparkTestUtils with TestTemplateWithTmpDir {
       logger.warn("test message 5")
 
     } finally {
-      logger.close
+      logger.close()
     }
 
-    val fs = FileSystems.getDefault()
+    val fs = FileSystems.getDefault
     assertTrue(Files.exists(fs.getPath(logFile)))
 
-    val lines = Source.fromFile(logFile).getLines.toArray
-    assertEquals(lines.size, 5)
+    val lines = Source.fromFile(logFile).getLines().toArray
+    assertEquals(lines.length, 5)
```

Comment:
@XianXing I'm curious about the rationale for both these changes. ```getLines``` is not side-effecting. Also is there a conventional preference for ```length``` over ```size```?